tests/files/c_source/abs.h

```diff
@@ -58,6 +58,63 @@
 #define ABS_BREMSE_50_FRAME_ID (0x560u)
 #define ABS_BREMSE_53_FRAME_ID (0x5c0u)
 
+#define ABS_BREMSE_33_FRAME_LEN (8u)
+#define ABS_BREMSE_10_FRAME_LEN (8u)
+#define ABS_BREMSE_11_FRAME_LEN (8u)
+#define ABS_BREMSE_12_FRAME_LEN (8u)
+#define ABS_BREMSE_13_FRAME_LEN (8u)
+#define ABS_DRS_RX_ID0_FRAME_LEN (8u)
+#define ABS_MM5_10_TX1_FRAME_LEN (8u)
+#define ABS_MM5_10_TX2_FRAME_LEN (8u)
+#define ABS_MM5_10_TX3_FRAME_LEN (8u)
+#define ABS_BREMSE_2_FRAME_LEN (8u)
+#define ABS_ABS_SWITCH_FRAME_LEN (8u)
+#define ABS_BREMSE_30_FRAME_LEN (8u)
+#define ABS_BREMSE_31_FRAME_LEN (8u)
+#define ABS_BREMSE_32_FRAME_LEN (8u)
+#define ABS_BREMSE_51_FRAME_LEN (8u)
+#define ABS_BREMSE_52_FRAME_LEN (8u)
+#define ABS_BREMSE_50_FRAME_LEN (8u)
+#define ABS_BREMSE_53_FRAME_LEN (8u)
+
+#define ABS_BREMSE_33_FRAME_EXTENDED (0)
```

Comment:
I suggest `#define ABS_BREMSE_33_IS_EXTENDED (0)` to have the same naming as the database message class.